src/components/Select/Select.js

```diff
@@ -364,4 +399,74 @@ class Select extends Component {
 
 export default Select;
 
+export const SelectComponentsPropTypes = {
+    ClearIndicator: PropTypes.oneOfType([PropTypes.element, PropTypes.func]),
+    Control: PropTypes.oneOfType([PropTypes.element, PropTypes.func]),
+    DropdownIndicator: PropTypes.oneOfType([PropTypes.element, PropTypes.func]),
+    DownChevron: PropTypes.oneOfType([PropTypes.element, PropTypes.func]),
+    CrossIcon: PropTypes.oneOfType([PropTypes.element, PropTypes.func]),
+    Group: PropTypes.oneOfType([PropTypes.element, PropTypes.func]),
+    GroupHeading: PropTypes.oneOfType([PropTypes.element, PropTypes.func]),
+    IndicatorsContainer: PropTypes.oneOfType([
+        PropTypes.element,
+        PropTypes.func
+    ]),
+    IndicatorSeparator: PropTypes.oneOfType([
+        PropTypes.element,
+        PropTypes.func
+    ]),
+    Input: PropTypes.oneOfType([PropTypes.element, PropTypes.func]),
+    LoadingIndicator: PropTypes.oneOfType([PropTypes.element, PropTypes.func]),
+    Menu: PropTypes.oneOfType([PropTypes.element, PropTypes.func]),
+    MenuList: PropTypes.oneOfType([PropTypes.element, PropTypes.func]),
+    MenuPortal: PropTypes.oneOfType([PropTypes.element, PropTypes.func]),
+    LoadingMessage: PropTypes.oneOfType([PropTypes.element, PropTypes.func]),
+    NoOptionsMessage: PropTypes.oneOfType([PropTypes.element, PropTypes.func]),
+    MultiValue: PropTypes.oneOfType([PropTypes.element, PropTypes.func]),
+    MultiValueContainer: PropTypes.oneOfType([
+        PropTypes.element,
+        PropTypes.func
+    ]),
+    MultiValueLabel: PropTypes.oneOfType([PropTypes.element, PropTypes.func]),
+    MultiValueRemove: PropTypes.oneOfType([PropTypes.element, PropTypes.func]),
+    Option: PropTypes.oneOfType([PropTypes.element, PropTypes.func]),
+    Placeholder: PropTypes.oneOfType([PropTypes.element, PropTypes.func]),
+    SelectContainer: PropTypes.oneOfType([PropTypes.element, PropTypes.func]),
+    SingleValue: PropTypes.oneOfType([PropTypes.element, PropTypes.func]),
+    ValueContainer: PropTypes.oneOfType([PropTypes.element, PropTypes.func])
+};
+
```

Comment:
Potremmo anche mettere un commento introduttivo prima dell'`export` e fare riferimento direttamente alla [documentazione di React Select](https://deploy-preview-2289--react-select.netlify.com/props#select-props), anziché ri-documentare qui.

Reply:
Si l'obiettivo era quello ma react-select utilizza TypeScript e non li esporta in JS (almeno non li ho trovati).